README.md

```diff
@@ -321,6 +322,9 @@ exclude = '''
 '''
 ```
 
+Note: `src` specifies a list of source files / directories which will be formatted
+(which can be specified as command line argument)
```

Comment:
I'm not so sure I like the wording here, perhaps: 
```suggestion
Tip: `src` specifies a list of default source files / directories which will be formatted
when _Black_ is run with no sources provided. Sources given on the command line
will override the list of sources in `pyproject.toml`.
```

(please wordsmith more to your liking - it might be a bit too long 😉 )